store/file/q4.go

```diff
@@ -102,17 +89,12 @@ func writeQ4(w io.Writer, eds *rsmt2d.ExtendedDataSquare) error {
 // of the File is read lazily.
 // If file is empty, the ErrEmptyFile is returned.
 // File must be closed after usage.
-func OpenQ4(path string) (*Q4, error) {
+func OpenQ4(path string, hdr *headerV0) (*Q4, error) {
```

Comment:
It feels wrong to pass private header. I propose to either:
* Make Header public, doesn't hurt.
* Make OpenQ4 private
  * Then stop implementing Accessor for it. It's not needed anymore as its not usable on its own.

Reply:
implemented option 2